### ISSUE TITLE
autoLoad stopped working in 5.1.0

### ISSUE THREAD

**Issue by martin-ecatch** [anthonyjgrove/react-google-login#292]
I just noticed that after upgrading from 5.0.7 to 5.1.0 the autoLoad property has no effect and it doesn't automatically open the google popup. I had to roll back to 5.0.7.
this is how I use it (* means redacted content):
```
        <GoogleLogin
          autoLoad
          clientId={********}
          buttonText="Login"
          onSuccess={*************}
          onFailure={**************}
          cookiePolicy={'single_host_origin'}
        />
```
Do I need to do anything else in the new version to enable autoLoad functionality or did it simply stop working?

Thank you for this great package btw!

**Comment by telmogoncalves** [comment]
I've opened a [PR](https://github.com/anthonyjgrove/react-google-login/pull/296/files) with a possible fix. If @anthonyjgrove agrees this should be fixed when merged 👍

**Comment by anthonyjgrove** [comment]
published v5.1.1

**Comment by martin-ecatch** [comment]
Legendary! Thanks for the quick fix!
